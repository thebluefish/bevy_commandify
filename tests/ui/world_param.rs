use bevy_commandify::*;

#[command]
fn foo(n_: usize) { }

/// Test that a command requires a `&mut World` fn parameter
fn main() { }