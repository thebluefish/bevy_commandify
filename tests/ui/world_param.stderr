error: Commands must take in a `&mut World` parameter
 --> tests/ui/world_param.rs:3:1
  |
3 | #[command]
  | ^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `command` (in Nightly builds, run with -Z macro-backtrace for more info)
